public/app/percona/backup/components/BackupInventory/AddBackupModal/AddBackupModal.service.ts

```diff
@@ -7,13 +7,23 @@ import { SelectableService } from './AddBackupModal.types';
 export const AddBackupModalService = {
   async loadServiceOptions(): Promise<Array<SelectableValue<SelectableService>>> {
     let result: Array<SelectableValue<SelectableService>> = [];
+    const supportedServices: Databases[] = [Databases.mysql, Databases.mongodb];
     const services = await InventoryService.getDbServices();
 
-    // TODO remove this constraint when more DB types are supported
-    if (services.mysql) {
-      result = services.mysql.map(
-        ({ id, name }): SelectableValue<SelectableService> => ({ label: name, value: { id, vendor: Databases.mysql } })
-      );
+    let serviceName: Databases;
+    for (serviceName in services) {
```

Comment:
```suggestion
    for (const serviceName in services) {
```

Reply:
By doing that, we're assuming `serviceName` as a string. Since `services` is of `DBServiceList` type (which is a map using `Databases` as key), that can't be done, because we can't use `string` as `Databases`

Reply:
(this is a way of solving this linter problem that we have on other places where we're forcibly using `any`)

Reply:
Got it, but anyway avoid explicit `for` loops unless strictly needed. In this case it looks to me that it can become `services.map((service: Databases) ...)` or `Object.keys(services).map` etc.

Furthermore always use `const` instead of `let`, unless `let` is really needed.

Reply:
copy that